code/modules/surgery/organs/filling_organ.dm

```diff
@@ -207,20 +207,22 @@
 				to_chat(owner, span_lovebold("My breasts should start lactating soon..."))
 		if(pregnantaltorgan) //there is no birthing so hopefully 2 hours for one stage is enough to last till round end, there is 0 to 3 belly sizes.
 			pre_pregnancy_size = pregnantaltorgan.organ_size
-			addtimer(CALLBACK(pregnantaltorgan, PROC_REF(handle_preggoness)), 2 HOURS, TIMER_STOPPABLE)
+			addtimer(CALLBACK(pregnantaltorgan, PROC_REF(handle_preggoness)), 30 MINUTES, TIMER_STOPPABLE)
 		else
 			pre_pregnancy_size = organ_size
-			addtimer(CALLBACK(src, PROC_REF(handle_preggoness)), 2 HOURS, TIMER_STOPPABLE)
+			addtimer(CALLBACK(src, PROC_REF(handle_preggoness)), 30 MINUTES, TIMER_STOPPABLE)
 
 /obj/item/organ/filling_organ/proc/handle_preggoness()
+	var/obj/item/organ/belly/bellyussy = owner.getorganslot(ORGAN_SLOT_BELLY)
+
 	var/datum/sprite_accessory/acc = accessory_type
 	to_chat(owner, span_lovebold("I notice my [src] has grown...")) //dont need to repeat this probably if size cant grow anyway.
 	if(organ_sizeable)
-		if(organ_size < 3)
-			organ_size += 1
+		if(bellyussy.organ_size < 3)
+			bellyussy.organ_size = pre_pregnancy_size + 1
```

Comment:
This makes no sense, it will just return the same number each cycle

Reply:
in theory, the stuff in red already does nothing, so I'm crossing my fingers and assuming it will continue to do nothing. I think there's like...some kind of issue where it's not picking up the organ, so I'm just changing the belly directly. I'll just comment it out

Reply:
Oh I see what you mean. I'll fix that.

Reply:
DONE!